test/python/test_mapper.py

```diff
@@ -200,6 +199,26 @@ def test_already_mapped(self):
 
         self.assertEqual(sorted(cx_qubits), [[3, 4], [3, 14], [5, 4], [9, 8], [12, 11], [13, 4]])
 
+    @requires_qe_access
+    def test_yzy_zyz_cases(self, QE_TOKEN, QE_URL, hub, group, project):
+        """Test mapper function yzy_to_zyz works in previously failed cases.
+
+        See: https://github.com/QISKit/qiskit-terra/issues/607
+        """
+        qiskit.wrapper.register(QE_TOKEN, QE_URL, hub, group, project)
+        if 'ibmqx4' in qiskit.wrapper.available_backends():
```

Comment:
why do you need to hardcode the backend here? also this is for compilation not execution, so i think just using `compile()` and passing your own `coupling_map` should work.

Reply:
I made a fake QX4 backend to test against.